content/blog/2022/10/2022-10-10-plugin-health-scoring-system-report.adoc

```diff
@@ -0,0 +1,130 @@
+---
+:layout: post
+:title: "Plugin Health Scoring System"
+:tags:
+- gsoc
+- gsoc2022
+- healthscore
+- probes
+- plugins
+:author: dheerajodha
+:sig: gsoc
+description: >
+  The Plugin Health Scoring System is a Google Summer of Code 2022 project.
+opengraph:
+  image: /images/gsoc/2022/plugin-health-score/introducing-the-plugin-health-scoring-system.png
+links:
+  gitter: https://gitter.im/jenkinsci/GSoC-Plugin_Health_Score
+---
+
+image::/images/gsoc/2022/plugin-health-score/introducing-the-plugin-health-scoring-system.png[GSoC, height=400, role=center, float=center]
+
+The goal of this blog is to showcase the work done during the Google Summer of Code 2022 coding phases.
+For a detailed description of the project, please see the link:/projects/gsoc/2022/projects/plugin-health-scoring-system/[project page].
+
+---
+
+== Overview
+
+. <<About Project>>
+. <<Architecture Diagram>>
+. <<Phase 1>>
+. <<Phase 2>>
+. <<Next Steps>>
+. <<Acknowledgments>>
+. <<Useful Links>>
+
+---
+
+== About Project
+
+Plugin maintenance is an extremely important phase of a plugin’s lifecycle. Jenkins has always been active in ongoing developments which raises a need for thousands of plugins to catch up with their latest developments, to stay useful for a broader set of users.
+
+We aim to fulfill this need by introducing a metric system to calculate the health score of Jenkins plugins. This health score measures the plugin’s development maturity level with the help of accurate numeric figures. This score attempts to provide an accurate picture of how much care and help a plugin needs and makes it easier for contributors to contribute. This score also allows users to make a conscious decision before installing and/or using a plugin.
+
+There were two coding phases in the GSoC 2022.
+
+---
+
+== Architecture Diagram
+
+image::/images/gsoc/2022/plugin-health-score/architecture-diagram.png[Architecture Diagram, height=690, role=center, float=center]
+
+---
+
+== Phase 1
+
+In this phase, we worked on setting up the project and followed best practices while doing so. The whole application is dockerized so that it is easier for anyone to spin up this project and test it out locally. There's continuous integration (CI) in place for the project to make sure that new changes are tested automatically. The useful details from the Jenkins Update Center are stored inside the database to run analysis.
```

Comment:
```suggestion
In this phase, we worked on setting up the project and followed best practices while doing so.
The whole application is dockerized so that it is easier for anyone to spin up this project and test it out locally.
There's continuous integration (CI) in place for the project to make sure that new changes are tested automatically.
The useful details from the Jenkins Update Center are stored inside the database to run analysis.
```
same formatting suggestion

Reply:
Done, thanks!